.github/workflows/llm_unit_tests_basic_windows.yml

```diff
@@ -0,0 +1,48 @@
+name: LLM Unit Tests Basic on Windows
+
+# Cancel previous runs in the PR when you push new commits
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+    paths:
+      - 'python/llm/**'
+      - '.github/workflows/llm_unit_tests_basic_windows.yml'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'python/llm/**'
+      - '.github/workflows/llm_unit_tests_basic_windows.yml'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  llm-unit-test-basic-windows:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest"]
+        python-version: ["3.8"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools==58.0.4
+          python -m pip install --upgrade wheel
+      - name: Run LLM-init test
+        shell: bash
+        run: |
+          bash python/llm/dev/release_default_windows.sh default false
+          pip install -i https://pypi.python.org/simple python/llm/dist/bigdl_llm-2.4.0.dev0-py3-none-win_amd64.whl
```

Comment:
may be we just `pip install -i https://pypi.python.org/simple python/llm/dist/*.whl` ? The version in filename might be deprecated soon.

Reply:
resolved